src/Microsoft.AspNetCore.Razor.OmniSharpPlugin/DefaultConfigurationProvider.cs

```diff
@@ -0,0 +1,222 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Composition;
+using System.Linq;
+using Microsoft.AspNetCore.Razor.Language;
+using Microsoft.Build.Execution;
+
+namespace Microsoft.AspNetCore.Razor.OmniSharpPlugin
+{
+    [Shared]
+    [Export(typeof(RazorConfigurationProvider))]
+    internal class DefaultConfigurationProvider : CoreProjectConfigurationProvider
```

Comment:
From the logic, it appears that `DefaultConfigurationProvider` is for >2.1, and `FallbackConfigurationProvider` is for <=2.1. The Default/Fallback naming doesn't imply versioning strongly to me - would names like Current vs Legacy be clearer? Or even indicate which side of the 2.2 boundary they are on explicitly in the name?

Reply:
Oops, should update the comment. Default is for >= 2.1 and Fallback is for < 2.1. As for naming, I brought them forward  from the Razor repo. Our project system impls which do similar things to these use those names; I think you're right though, we can do better 👍